accounts/abi/bind/precompilebind/precompile_contract_template.go

```diff
@@ -107,6 +107,24 @@ var (
 	}
 {{- end}}
 
+{{range .Contract.Events}}
+	{{$event := .}}
+	{{$createdStruct := false}}
+	{{- range .Normalized.Inputs}}
+	{{- if not .Indexed}}
```

Comment:
nit: I feel using an early return would be more readable. Something like
```
{{- if .Indexed}}
	{{ continue }}
{{- end}}
```
would remove one of the `{- end}` within the loop.